ddtrace/tracer/tracer.go

```diff
@@ -426,6 +426,11 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 		if t.config.universalVersion || (!t.config.universalVersion && span.Service == t.config.serviceName) {
 			span.setMeta(ext.Version, t.config.version)
 		}
+		// If the span has a different service than the global service, attribute and set the config version as the
```

Comment:
Hmm What purpose does this serve here? Also I'm not sure the comment makes sense? Should the comma there be dropped?

Reply:
I updated that comment to hopefully make things clearer but I could use your knowledge to make sure that's all correct. The main reason this exists is that the version wasn't set for database spans because database spans have a database-inferred service name that's set on them. Instead of just blindly setting the service version as the version on the database span, I thought it would be better to name this one ParentVersion since that's the version of the parent. 

Does this still make sense? If there's a better way to think about the injection of the calling service's version on sql spans, I'm interested to hear.

Reply:
This ended up being obsolete with the latest changes.